docs/development/build-instructions-windows.md

```diff
@@ -1,29 +1,30 @@
-# Build instructions (Windows)
+# Build Instructions (Windows)
+
+Follow the guidelines below for building Electron on Windows.
 
 ## Prerequisites
 
 * Windows 7 / Server 2008 R2 or higher
 * Visual Studio 2013 - [download VS 2013 Community Edition for
-  free](https://www.visualstudio.com/downloads/download-visual-studio-vs)
+  free](https://www.visualstudio.com/downloads/download-visual-studio-vs).
 * [Python 2.7](http://www.python.org/download/releases/2.7/)
 * [Node.js](http://nodejs.org/download/)
-* [git](http://git-scm.com)
+* [Git](http://git-scm.com)
 
-If you don't have a Windows installation at the moment,
-[modern.ie](https://www.modern.ie/en-us/virtualization-tools#downloads) has
-timebombed versions of Windows that you can use to build Electron.
+If you don't currently have a Windows installation, [modern.ie](https://www.modern.ie/en-us/virtualization-tools#downloads)
+has timebombed versions of Windows that you can use to build Electron.
 
-The building of Electron is done entirely with command-line scripts, so you
-can use any editor you like to develop Electron, but it also means you can
-not use Visual Studio for the development. Support of building with Visual
-Studio will come in the future.
+Building Electron is done entirely with command-line scripts and cannot be done
+with Visual Studio. You can develop Electron with any editor but support for
+building with Visual Studio will come in the future.
```

Comment:
My sense was the previously "development" was used when "building" was intended. I reworded the sentence and want to double check I didn't change the meaning @zcbenz